extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -201,6 +204,10 @@ const stateComparisonFunctions = {
  */
 const actions = (state, action, data) => {
   switch (action) {
+    case Action.ADD_NEW_PAGE_ID:
+      return /** @type {!State} */ (Object.assign({}, state, {
+        [StateProperty.NEW_PAGE_ADDED_ID]: data,
+      }));
```

Comment:
What do you think about going to extra mile, and making this an array of page IDs?

We'd store the page IDs in the store, and the system layer code would use that to build the first time, and then react to when pages are added through the live list.
It'd allow you to add pages anywhere in the story if needed. With the implementation from the PR, you can only add one page at the end, but with an array you could remove / insert pages at any positions, which is more future proof.
Also, it'd more general purpose information about the story, which is more aligned with what the store is doing today. Knowing what was the latest page added is not very interesting outside of this livelist feature.

It'd require a small refactoring to no longer accept `pageIds` when building the system layer, but not having to pass this array across 3 levels of components (lasagna code) would be really cool.

Reply:
Done.